app/views/case_contacts/_contact_types.html.erb

```diff
@@ -14,7 +14,7 @@
       <% contact_type_groups.each do |group| %>
         <td class="align-top d-inline-block pr-5 pl-5 pb-4">
           <h5 class="mb-2"> <%= group.name %> </h5>
-          <% group.contact_types.alphabetically.filter do |ct|
+          <% group.contact_types.alphabetically do |ct| #removes filter
```

Comment:
please add a test :)

Reply:
@compwron Test added to this open PR! Please let me know if anything else needs attention, thank you :)

Reply:
If the `filter` is being removed, then don't we need to remove the `do/end` block as well? That was the block that was being passed to the filter, right?

Reply:
@gaurijo the alphabetically method doesn't take a block, so you should be able to remove the `do/end` block. It was there for the `filter` method.